nekoyume/Assets/_Scripts/UI/Module/TextButton.cs

```diff
@@ -1,83 +1,34 @@
-using System;
-using System.Collections.Generic;
-using Nekoyume.Game.Controller;
 using UnityEngine;
-using UnityEngine.UI;
+using UnityEngine.UI;
+using TMPro;
 
 namespace Nekoyume.UI.Module
-{
-    using UniRx;
-
+{
+    [RequireComponent(typeof(Button))]
     public class TextButton : MonoBehaviour
     {
-        public Button button;
-        public Image backgroundImage;
-        public Text text;
-        
-        private Model.TextButton _data;
+        [SerializeField]
+        private TextMeshProUGUI text = null;
         
-        private readonly List<IDisposable> _disposablesForAwake = new List<IDisposable>();
-        private readonly List<IDisposable> _disposablesForSetData = new List<IDisposable>();
+        [SerializeField]

```

Comment:
It seems can remove `SerializeField` attribute from the `button` if the `button` assigned in `Awake()` with `RequiredComponent(typeof(Button))` attribute.